### DISCUSSION TITLE
[Enhancement]: Add Pull request validation for linked issue and milestone

### DISCUSSION THREAD

**Discussion by PropmanProphecy** [microsoft/AL-Go#1602] [Ideas]
### Feature description

since the BusinessCentralApps repo have check to see if pull request have an issue and milestone linked.
would it be possible to also add some similar functionality to AL-GO

**Comment by basprintvis** [comment]
Any update on this enhancement request?

**Reply by freddydk** [reply]
We will be going through all feature request and try to prioritize these over the next weeks.

**Comment by mazhelez** [comment]
Meanwhile, feel free to re-use the functionality we came up with for _BCApps_ and _BusinessCentralApps_.

E.g. https://github.com/microsoft/BCApps/blob/main/.github/workflows/WorkitemValidation.yaml
